data/0.95.0/schema/ciphertext_election_context.schema.json

```diff
@@ -1,31 +1,22 @@
 {
-  "$schema": "http://json-schema.org/draft-07/schema",
```

Comment:
These should be in their own folder `data/1.0.0-preview-1/schema` with their own sample data.